packages/victory-core/src/victory-util/add-events.js

```diff
@@ -274,11 +275,14 @@ export default (WrappedComponent, options) => {
       const parentProps = isContainer
         ? this.getComponentProps(component, "parent", "parent")
         : {};
+
       if (parentProps.events) {
         this.globalEvents = Events.getGlobalEvents(parentProps.events);
         parentProps.events = Events.omitGlobalEvents(parentProps.events);
       }
-      return React.cloneElement(component, parentProps, children);
+      
+      const componentProps = { ...parentProps, ...parentProps.userProps };
```

Comment:
Where is `parentProps.userProps` coming from? Is this outdated, or are we still setting this somewhere in a `getComponentProps` function?